src/exercism/clojure_analyzer.clj

```diff
@@ -1,10 +1,31 @@
 (ns exercism.clojure-analyzer
   (:require
    [clojure.tools.analyzer.jvm :as ana.jvm]
+   [clj-kondo.core :as clj-kondo]
    [cheshire.core :as json]
-   [clojure.edn :as edn])
+   [clojure.edn :as edn]
+   [rewrite-clj.zip :as z])
   (:gen-class))
 
+;; load example two-fer solution, parse code to nodes, 
+;; create a zipper, and navigate to the first non-whitespace node
+(def zloc (z/of-file "resources/example.clj"))
```

Comment:
Is this test code? As in: does the analyzer right now always work on the example two-fer solution?

Reply:
Yes that's just me experimenting with these zipper data structures for navigating trees. It may or may not end up being used, depending on what kind of information we need to know.

Reply:
Sounds good.